project-words.txt

```diff
@@ -681,6 +681,8 @@ Postbots
 permissionlessly
 redcircle
 bluecircle
+Degen
```

Comment:
can you put these in alphabetical order? I know that none of the others are at the bottom, but if you can put it towards the top where it is alphabetical, that would be awesome @toshitsom

Reply:
I have arranged the works please check